NEWS.md

```diff
@@ -2,6 +2,40 @@
 * Updated association profiles samples. 
 * Updated API endpoint for getting details by master id. Added query param to specify return type. Added searching by cql query for details
 * Updated API endpoint for getting masters by detail id. Added query param to specify return type. Added searching by cql query for details
+* Updated API endpoints for Profile Association entity.
```

Comment:
Could you please specify what exactly was done here. For example added these ... params and etc.
Guys @OleksiiKuzminov @Igor-Gorchakov @KaterynaSenchenko is it necessary to copy paste whole table?
Maybe it will be enough just put endpoints those were changed

Reply:
@Anatolii-Starkov, we usually copy the whole table in order to have an account of all relevant endpoints provided by the latest API, it is proved to be convenient when we release the modules. So I suggest to follow our convention and leave the table in the news as is.